caffe2/serialize/inline_container.cc

```diff
@@ -556,6 +556,17 @@ void PyTorchStreamWriter::writeEndOfFile() {
       writeRecord("version", version.c_str(), version.size());
     }
   }
+
+  // If no "byteorder" record in the output model, rewrites byteorder info
+  if(allRecords.find("byteorder") == allRecords.end()) {
```

Comment:
Seems like some tests depend on existing behavior so writing this is breaking them (see https://github.com/pytorch/pytorch/blob/main/test/package/test_misc.py#L27 and https://hud.pytorch.org/pr/101973